docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -76,20 +74,164 @@ The bundling strategy was taken care of by a Create React App, and no locale was
 The results may vary in your application depending on the version of each library, the locale, and the bundling strategy used.
 :::
 
+### Other peer dependencies
+
+The Date and Time Pickers package has a peer dependency on `@mui/material`.
+If you are not already using it in your project, you can install it with:
+
+```sh
+// with npm
+npm install @mui/material @emotion/react @emotion/styled
+
+// with yarn
+yarn add @mui/material @emotion/react @emotion/styled
+```
+
+<!-- #react-peer-version -->
```

Comment:
Does it have an interest for maintenance of the docs?
```suggestion
```

Reply:
No idea, I copy pasted the grid doc :laughing: